.storybook/preview.js

```diff
@@ -27,3 +28,12 @@ for (let i = -3; i <= 6; i++) {
 }
 addDecorator(withPaddings);
 addParameters({ paddings });
+/**
+ * Default viewport is Responsive: 100% x 100%. When a breakpoint is chosen, this
+ * will show as the Reset Viewport option
+ */
```

Comment:
It's unclear to me what this comment is trying to convey relative to the adjacent code.

Reply:
Sorry, no it's not necessarily obvious. We don't set this viewport, but it's the stock default, and I thought it would be good to indicate that somewhere. In other words, if we don't actually set the default, this is what it will be on page load (100% x 100%). If I rephrased slightly, would it make sense, and do you think it's worth adding?

```